src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -208,13 +593,55 @@ public double GetDouble()
             throw new FormatException();
         }
 
+        /// <summary>
+        ///   Attempts to represent the current JSON number as a <see cref="float"/>.
+        /// </summary>
+        /// <param name="value">Receives the value.</param>
+        /// <remarks>
+        ///   This method does not parse the contents of a JSON string value.
+        ///
+        ///   On .NET Core this method does not return <c>false</c> for values larger than
```

Comment:
Should we use `<see langword="true"/>` instead of `<c></c>`?

Reply:
Here and elsewhere we have true/false in the comments.

Reply:
I changed all of them except the ones in JsonValueType, since they're about the JSON notion of true/false/null, vs the C# one.